.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 5 days'
+          close-issue-message: "This issue was closed because it has been inactive for 14 days since being marked as stale."
```

Comment:
"was closed due to inactivity". I don't think the 14 days claim is correct given the stale issue message above.

Reply:
Right. Makes sense.

```suggestion
          close-issue-message: "This issue was closed because it has been inactive for 19 days since being marked as stale."
```